senza/manaus/cloudformation.py

```diff
@@ -124,6 +124,11 @@ def resources(self) -> Iterator:
         for resource in resources:
             resource_type = resource["ResourceType"]
             if resource_type == ResourceType.route53_record_set:
+                physical_resource_id = resource.get('PhysicalResourceId')
+                if physical_resource_id is None:
```

Comment:
Some resources with delete_complete have a PhysicalResourceId. From the same stack trace:

    {'LastUpdatedTimestamp': datetime.datetime(..), 
     'LogicalResourceId': 'xxxLBSecurityGroup', 
     'PhysicalResourceId': 'xxx-xxx-xx-xxxBSecurityGroup-xxxxxx', 
     'ResourceStatus': 'DELETE_COMPLETE', 
     'ResourceType': 'AWS::EC2::SecurityGroup' }

Granted none of them are Route53 records but I couldn't find documentation on this.